### PR TITLE
Update dumpling-overview.md

### PR DESCRIPTION
**Pull request by Frank945946**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dumpling-overview.md [hunk 1]
```diff
@@ -404,6 +404,7 @@ SET GLOBAL tidb_gc_life_time = '10m';
...
 | `--csv-line-terminator`      | The terminator at the end of a line for CSV files. When exporting data to a CSV file, you can specify the desired terminator with this option. This option supports "\\r\\n" and "\\n". The default value is "\\r\\n", which is consistent with the earlier versions. Because quotes in bash have different escaping rules, if you want to specify LF (linefeed) as a terminator, you can use a syntax similar to `--csv-line-terminator $'\n'`. | "\\r\\n" |
+| `--csv-output-dialect`          | Dumpling will represent binary data types with UTF-8 format in CSV files by default. To export binary data in another format like HEX, set the parameter to 'redshift', but the '0x' prefix will be omitted, such as '0x61' being represented as '61'.                                                                                                                                                                                                                                                                  | ""                                      |
```
Comment (hfxsd) [inline dumpling-overview.md:407]:
```suggestion
| `--csv-output-dialect`      | Dumpling represents binary data types in CSV files in UTF-8 format by default. To export binary data in another format such as HEX, set the parameter to 'redshift'. Note that the '0x' prefix will be removed. For example, '0x61' is represented as '61'.  | ""  |
```

File: dumpling-overview.md [hunk 2]
```diff
@@ -404,6 +404,7 @@ SET GLOBAL tidb_gc_life_time = '10m';
...
 | `--csv-null-value`           | Representation of null values in CSV files                                                                                                                                                                                                                                                                                         | "\\N"                                      |
 | `--csv-line-terminator`      | The terminator at the end of a line for CSV files. When exporting data to a CSV file, you can specify the desired terminator with this option. This option supports "\\r\\n" and "\\n". The default value is "\\r\\n", which is consistent with the earlier versions. Because quotes in bash have different escaping rules, if you want to specify LF (linefeed) as a terminator, you can use a syntax similar to `--csv-line-terminator $'\n'`. | "\\r\\n" |
+| `--csv-output-dialect`      | Dumpling represents binary data types in CSV files in UTF-8 format by default. To export binary data in another format such as HEX, set the parameter to 'redshift'. Note that the '0x' prefix will be removed. For example, '0x61' is represented as '61'.  | ""  |
```
Comment (qiancai) [inline dumpling-overview.md:407]:
```suggestion
| `--csv-output-dialect`      | Dumpling represents binary data types in CSV files in UTF-8 format by default. To export binary data in another format such as HEX, set the parameter to `"redshift"`. Note that the `0x` prefix will be removed. For example, `0x61` is represented as `61`.  | ""  |
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

File: dumpling-overview.md [hunk 3]
```diff
@@ -404,6 +404,7 @@ SET GLOBAL tidb_gc_life_time = '10m';
...
 | `--csv-line-terminator`      | The terminator at the end of a line for CSV files. When exporting data to a CSV file, you can specify the desired terminator with this option. This option supports "\\r\\n" and "\\n". The default value is "\\r\\n", which is consistent with the earlier versions. Because quotes in bash have different escaping rules, if you want to specify LF (linefeed) as a terminator, you can use a syntax similar to `--csv-line-terminator $'\n'`. | "\\r\\n" |
+| `--csv-output-dialect`      | The value can be 'snowflake', 'redshift', or 'bigquery', indicating that the source data can be exported to CSV in the format required by the aforementioned databases. Such as binary data types will be converted to hexadecimal, but the '0x' prefix will be omitted. For example, '0x61' will be represented as '61'  | ""  |
```
Comment (hfxsd) [inline dumpling-overview.md:407]:
```suggestion
| `--csv-output-dialect`      | Indicates that the source data can be exported to a CSV file in a specific required format for the database. The option value can be `"snowflake"`, `"redshift"`, and `"bigquery"`. For example, binary data types will be converted to hexadecimal, but the '0x' prefix will be removed. For example, '0x61' will be represented as '61'  | ""  |
```

**Review by Lloyd-Pottiger — APPROVED** [review]

File: dumpling-overview.md [hunk 4]
```diff
@@ -404,6 +404,7 @@ SET GLOBAL tidb_gc_life_time = '10m';
...
 | `--csv-line-terminator`      | The terminator at the end of a line for CSV files. When exporting data to a CSV file, you can specify the desired terminator with this option. This option supports "\\r\\n" and "\\n". The default value is "\\r\\n", which is consistent with the earlier versions. Because quotes in bash have different escaping rules, if you want to specify LF (linefeed) as a terminator, you can use a syntax similar to `--csv-line-terminator $'\n'`. | "\\r\\n" |
+| `--csv-output-dialect`      | Indicates that the source data can be exported to a CSV file in a specific required format for the database. The option value can be `"snowflake"`, `"redshift"`, and `"bigquery"`. For example, if you set the option to `"snowflake"` or `"redshift"`, the binary data type will be converted to hexadecimal, but the `0x` prefix will be removed. For example, `0x61` will be represented as `61`. If you set the option to `"bigquery"`, the binary data type will be encoded using base64.  | ""  |
```
Comment (hfxsd) [inline dumpling-overview.md:407]:
```suggestion
| `--csv-output-dialect`      | Indicates that the source data can be exported to a CSV file in a specific required format for the database. The option value can be `"snowflake"`, `"redshift"`, and `"bigquery"`. If you set the option to `"snowflake"` or `"redshift"`, the binary data type will be converted to hexadecimal, but the `0x` prefix will be removed. For example, `0x61` will be represented as `61`. If you set the option to `"bigquery"`, the binary data type will be encoded using base64.  | ""  |
```

File: dumpling-overview.md [hunk 5]
```diff
@@ -404,6 +404,7 @@ SET GLOBAL tidb_gc_life_time = '10m';
...
 | `--csv-line-terminator`      | The terminator at the end of a line for CSV files. When exporting data to a CSV file, you can specify the desired terminator with this option. This option supports "\\r\\n" and "\\n". The default value is "\\r\\n", which is consistent with the earlier versions. Because quotes in bash have different escaping rules, if you want to specify LF (linefeed) as a terminator, you can use a syntax similar to `--csv-line-terminator $'\n'`. | "\\r\\n" |
+| `--csv-output-dialect`      | Indicates that the source data can be exported to a CSV file in a specific required format for the database. The option value can be `"snowflake"`, `"redshift"`, and `"bigquery"`. If you set the option to `"snowflake"` or `"redshift"`, the binary data type will be converted to hexadecimal, but the `0x` prefix will be removed. For example, `0x61` will be represented as `61`. If you set the option to `"bigquery"`, the binary data type will be encoded using base64.  | ""  |
```
Comment (hfxsd) [inline dumpling-overview.md:407]:
```suggestion
| `--csv-output-dialect`      | Indicates that the source data can be exported to a CSV file in a specific required format for the database. The option value can be `"snowflake"`, `"redshift"`, and `"bigquery"`. If you set the option to `"snowflake"` or `"redshift"`, the binary data type will be converted to hexadecimal, but the `0x` prefix will be removed. For example, `0x61` will be represented as `61`. If you set the option to `"bigquery"`, the binary data type will be encoded using base64. The default value is null, which means to encode and export the data according to UTF-8. In some cases, the binary string will be garbled. | ""  |
```

File: dumpling-overview.md [hunk 6]
```diff
@@ -404,6 +404,7 @@ SET GLOBAL tidb_gc_life_time = '10m';
...
 | `--csv-line-terminator`      | The terminator at the end of a line for CSV files. When exporting data to a CSV file, you can specify the desired terminator with this option. This option supports "\\r\\n" and "\\n". The default value is "\\r\\n", which is consistent with the earlier versions. Because quotes in bash have different escaping rules, if you want to specify LF (linefeed) as a terminator, you can use a syntax similar to `--csv-line-terminator $'\n'`. | "\\r\\n" |
+| `--csv-output-dialect`      | Indicates that the source data can be exported to a CSV file in a specific required format for the database. The option value can be `"snowflake"`, `"redshift"`, and `"bigquery"`. If you set the option to `"snowflake"` or `"redshift"`, the binary data type will be converted to hexadecimal, but the `0x` prefix will be removed. For example, `0x61` will be represented as `61`. If you set the option to `"bigquery"`, the binary data type will be encoded using base64. The default value is null, which means to encode and export the data according to UTF-8. In some cases, the binary string will be garbled. | ""  |
```
Comment (hfxsd) [inline dumpling-overview.md:407]:
```suggestion
| `--csv-output-dialect`      | Indicates that the source data can be exported to a CSV file in a specific required format for the database. The option value can be `"snowflake"`, `"redshift"`, and `"bigquery"`. If you set the option to `"snowflake"` or `"redshift"`, the binary data type will be converted to hexadecimal, but the `0x` prefix will be removed. For example, `0x61` will be represented as `61`. If you set the option to `"bigquery"`, the binary data type will be encoded using base64. The default value is null, which means to encode and export the data according to UTF-8. In some cases, the binary string will be garbled. | `""`  |
```

**Review by hfxsd — APPROVED** [review]

File: dumpling-overview.md [hunk 7]
```diff
@@ -404,6 +404,7 @@ SET GLOBAL tidb_gc_life_time = '10m';
...
 | `--csv-line-terminator`      | The terminator at the end of a line for CSV files. When exporting data to a CSV file, you can specify the desired terminator with this option. This option supports "\\r\\n" and "\\n". The default value is "\\r\\n", which is consistent with the earlier versions. Because quotes in bash have different escaping rules, if you want to specify LF (linefeed) as a terminator, you can use a syntax similar to `--csv-line-terminator $'\n'`. | "\\r\\n" |
+| `--csv-output-dialect`      | Indicates that the source data can be exported to a CSV file in a specific required format for the database. The option value can be `"snowflake"`, `"redshift"`, and `"bigquery"`. If you set the option to `"snowflake"` or `"redshift"`, the binary data type will be converted to hexadecimal, but the `0x` prefix will be removed. For example, `0x61` will be represented as `61`. If you set the option to `"bigquery"`, the binary data type will be encoded using base64. The default value is null, which means to encode and export the data according to UTF-8. In some cases, the binary string will be garbled. | `""`  |
```
Comment (hfxsd) [inline dumpling-overview.md:407]:
```suggestion
| `--csv-output-dialect`      | Indicates that the source data can be exported to a CSV file in a specific required format for the database. The option value can be `"snowflake"`, `"redshift"`, and `"bigquery"`. If you set the option to `"snowflake"` or `"redshift"`, the binary data type will be converted to hexadecimal, but the `0x` prefix will be removed. For example, `0x61` will be represented as `61`. If you set the option to `"bigquery"`, the binary data type will be encoded using base64. The default value is null, which means to encode and export the data according to UTF-8. In some cases, the binary string might contain garbled characters. | `""`  |
```

File: dumpling-overview.md [hunk 8]
```diff
@@ -404,6 +404,7 @@ SET GLOBAL tidb_gc_life_time = '10m';
...
 | `--csv-line-terminator`      | The terminator at the end of a line for CSV files. When exporting data to a CSV file, you can specify the desired terminator with this option. This option supports "\\r\\n" and "\\n". The default value is "\\r\\n", which is consistent with the earlier versions. Because quotes in bash have different escaping rules, if you want to specify LF (linefeed) as a terminator, you can use a syntax similar to `--csv-line-terminator $'\n'`. | "\\r\\n" |
+| `--csv-output-dialect`      | Indicates that the source data can be exported to a CSV file in a specific required format for the database. The option value can be `"snowflake"`, `"redshift"`, and `"bigquery"`. If you set the option to `"snowflake"` or `"redshift"`, the binary data type will be converted to hexadecimal, but the `0x` prefix will be removed. For example, `0x61` will be represented as `61`. If you set the option to `"bigquery"`, the binary data type will be encoded using base64. The default value is null, which means to encode and export the data according to UTF-8. In some cases, the binary string might contain garbled characters. | `""`  |
```
Comment (qiancai) [inline dumpling-overview.md:407]:
```suggestion
| `--csv-output-dialect`      | Indicates that the source data can be exported to a CSV file in a specific required format for the database. The option value can be `""`, `"snowflake"`, `"redshift"`, or `"bigquery"`.  The default value is `""`, which means to encode and export the source data according to UTF-8. If you set the option to `"snowflake"` or `"redshift"`, the binary data type in the source data will be converted to hexadecimal, but the `0x` prefix will be removed. For example, `0x61` will be represented as `61`. If you set the option to `"bigquery"`, the binary data type will be encoded using base64. In some cases, the binary strings might contain garbled characters. | `""`  |
```

**Pull request merged**